avogadro/qtgui/moleculemodel.cpp

```diff
@@ -16,19 +16,31 @@ namespace Avogadro::QtGui {
 MoleculeModel::MoleculeModel(QObject* p)
   : QAbstractItemModel(p), m_activeMolecule(nullptr)
 {
+  const QPalette defaultPalette;
+  bool darkMode = (defaultPalette.color(QPalette::WindowText).lightness() >
+                   defaultPalette.color(QPalette::Window).lightness());
+  loadIcons(darkMode);
 }
 
 QModelIndex MoleculeModel::parent(const QModelIndex&) const
 {
   return QModelIndex();
 }
 
+void MoleculeModel::loadIcons(bool darkMode)
+{
+  QString iconPath = ":icons/fallback/32x32/";
+  QString plusIconPath = iconPath + (darkMode ? "plus-dark.png" : "plus-light.png");
+
+  m_plusIcon = QIcon::fromTheme("file-add", QIcon(plusIconPath));
```

Comment:
I don't think "file-add" is a thing. Do you want "document-new" instead?
It would be okay to just go with the plusIconPath, but "file-add" seems like it's always going to fail.